pages/about/contributing-in-github.md

```diff
@@ -0,0 +1,119 @@
+
+- You can always return to your opened issue by going to the[issues section](https://github.com/elixir-europe/rdmkit/issues) of our GitHub repo.  \
+-->
+
+- Find where to edit on GitHub
+
+- If you want to contribute to an existing chapter, go to the chapter on the site and click the **Edit me** pencil icon next to the page title.
+
+![](/assets/img/6U1_Image_3.png)
+
+![](/assets/img/hUE_Image_4.png)
+
+- Copy content of a template into new page.
+
+- You will be taken to the correct GitHub repository, where you will look for a pencil icon on the top right.
+
+![](/assets/img/oCM_Image_5.png)
+
+<!--
+
+## **Linking resources and other pages (optional)**
+
+- If you have mentioned tools or resources in your text, you will have to add them to the[ ](https://rdmkit.elixir-europe.org/tool_resource_update)[tool and resource list](https://rdmkit.elixir-europe.org/tool_resource_update).
+
+- If you want to list training material or link to other RDMkit pages, add it to the page metadata. Read more on how to do this in our[ ](https://rdmkit.elixir-europe.org/page_metadata)[page metadata guide](https://rdmkit.elixir-europe.org/page_metadata).
+
+**Important:** In general terms, you must avoid manual interlinking of RDMkit pages.
+
+-->
+
+## **Submit your changes**
+
+1. When you are done with your changes, commit them by clicking the green button at the top right corner.
+
+![](/assets/img/UTC_Image_6.png)
+
+2. This directs you to the **Propose changes** section where you need to write a title and a brief explanation of your changes.
+
+![](/assets/img/fH5_Image_7.png)
+
+3. Click **Propose changes**.
+
+![PrDraft pull request on GitHub](/assets/img/Gfi-prdraft-pull-request-github.png)
+
+![Linking issues in a pull request on GitHub](/assets/img/CI7-linking-issues-pull-request-github.png)
+
+**Tip:** You can create a draft pull request when you?re not ready to submit and will need to work more on it later. You can find more information about draft pull requests in the [GitHub documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests)
+
+**Tip:** You can return to your pull request by going to the [pull request section](https://github.com/elixir-europe/rdmkit/pulls) of our GitHub repo.
+
+6. Repository admin will accept your pull request unlessthey have comments that should be resolved before acceptance. In such case you will be notified by e-mail. To address comments, you must edit your pull request.
+
+1. Go to your pull request. You can return to it by going to the[pull request section](https://github.com/elixir-uk/elixir-ds-handbook/pulls) of our GitHub repo
+
+![Files changed tab on GitHub](/assets/img/FOL-files-changed-tab-github.png)
+
+3. Click on the icon with 3 dots ??? of the file you want to edit and then click on ?Edit file?.
```

Comment:
Instead of "the icon with 3 dots ???", you can say "Click the three dots i.e. the ellipses (...) in the top right hand corner..."

Reply:
Fixed